### DISCUSSION TITLE
Naming for "unofficial" crates

### DISCUSSION THREAD

**Discussion by marknefedov** [trillium-rs/trillium#85] [General]
I'm planning to publish some handler in the near future and don't want to hijack your trillium-handler-name pattern (if you will release crates with similar names), what would be preferable way to indicate crate was created to be used with trillium?

**Comment by jbr** [comment]
Thanks for using trillium and writing a crate for it!

`trillium-descriptive-name` seems great to me! My intent is for trillium to be an ecosystem eventually, and for the trillium-… crates that I've written to just be a subset of the ecosystem. I'd also love to include any other trillium crates in the guide-level documentation alongside the current crate, with permission and authorship attributed of course.

Please let me know if there are any apis that are missing or confusing — supporting library authors is a very high priority. Especially early on, I'll send PRs to accommodate any breaking changes (eg #51/#64) to the `trillium` or `trillium-http` crates in order to keep the burden on ecosystem library authors low.
